lib/trailblazer/pro/session.rb

```diff
@@ -60,9 +60,9 @@ module DSL
       def trace_operations(operation_hash)
         decisions =
           if operation_hash == :all
-            Trailblazer::Operation.extend(Trailblazer::Pro::Operation::Call)
+            Trailblazer::Operation.extend(Trailblazer::Pro::Operation::Call) # extend all OPs.
 
-            raise "implement me"
+            [->(*) { [Trace::Wtf, {}] }]
```

Comment:
Awesome!!! :sun_with_face: :coffee: but we need to add a test for that, unfortunately...